modules/nf-core/csvtk/concat/main.nf

```diff
@@ -4,11 +4,11 @@ process CSVTK_CONCAT {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/csvtk:0.30.0--h9ee0642_0' :
-        'biocontainers/csvtk:0.30.0--h9ee0642_0' }"
+        'https://depot.galaxyproject.org/singularity/csvtk:0.31.0--h9ee0642_0' :
+        'biocontainers/csvtk:0.31.0--h9ee0642_0' }"
 
     input:
-    tuple val(meta), path(csv)
+    tuple val(meta), path(csv, name: 'inputs/csv*/*')
```

Comment:
Why the double star here? Does that make them eg inputs/csv1/1.csv?

Reply:
No, the second `*` preserves the [source filename](https://www.nextflow.io/docs/latest/process.html#multiple-input-files), resulting in something like `inputs/csv1/original_name.csv`. Personally, I'm fine with all the files being symlinked using new names (e.g., `csv1.csv`, `csv2.csv`, ...). However, I considered that others using this module might prefer to see the original filenames in the command, which is why I implemented it this way.

Reply:
Ok, fine, I always have to look up the patterns.